lke_node_pools.go

```diff
@@ -33,6 +33,16 @@ type LKENodePoolLinode struct {
 	Status     LKELinodeStatus `json:"status"`
 }
 
+// LKENodePoolTaint represents a corev1.Taint to add to an LKENodePool
+type LKENodePoolTaint struct {
+	Key    string `json:"key"`
+	Value  string `json:"value,omitempty"`
+	Effect string `json:"effect"`
```

Comment:
> Effect: The effect value must be NoSchedule, PreferNoSchedule, or NoExecute.

I'm reading the API doc and find the effect value is one of the above. Could you build `Effect` to be an enum type?

Reply:
Sure, I've updated this.